src/OrchardCore.Modules/OrchardCore.Localization/ModularPoFileLocationProvider.cs

```diff
@@ -60,7 +60,7 @@ public IEnumerable<IFileInfo> GetLocations(string cultureName)
                 yield return new PhysicalFileInfo(new FileInfo(PathExtensions.Combine(_applicationDataContainer, _resourcesContainer, extension.Id, poFileName)));
 
                 // \src\OrchardCore.Cms.Web\App_Data/Localization/OrchardCore.Cms.Web-fr-CA.po
-                yield return new PhysicalFileInfo(new FileInfo(PathExtensions.Combine(_applicationDataContainer, _resourcesContainer, extension.Id + "." + poFileName)));
+                yield return new PhysicalFileInfo(new FileInfo(PathExtensions.Combine(_applicationDataContainer, _resourcesContainer, extension.Id + "-" + poFileName)));
```

Comment:
Any chance to move that to a const and call it `CultureDelimiter` ?

Reply:
Instead of replacing it just add the option to use either a "-" or a "."
Maybe more confusing, that way. If we find so then let's stick to Crowdin specifications.
I'm surprised though that Crowdin works that way.